upgrading/topics/keycloak/changes.adoc

```diff
@@ -1,5 +1,16 @@
 == Migration Changes
 
+=== Migrating to 16.0.0
+
+==== Proxy environment variables
+
+{project_name} now respects the standard `HTTP_PROXY`, `HTTPS_PROXY` and `NO_PROXY` environment variables for outgoing
+HTTP requests. This could lead to unexpected use of proxy server if you have for example `HTTP_PROXY` variable defined but have no
```

Comment:
This situation -- always try to add a noun after "this" to be clear what "this is referring to."

Reply:
Done.